modules/gapi/perf/common/gapi_core_perf_tests_inl.hpp

```diff
@@ -1781,7 +1991,10 @@ PERF_TEST_P_(ConcatVertPerfTest, TestPerformance)
     }
 
     // Comparison ////////////////////////////////////////////////////////////
-    EXPECT_EQ(0, cv::norm(out_mat_ocv, out_mat_gapi, NORM_INF));
+    {
+        EXPECT_TRUE(cmpF(out_mat_ocv, out_mat_gapi));
+        EXPECT_EQ(0, cv::norm(out_mat_ocv, out_mat_gapi, NORM_INF));
```

Comment:
Please leave only one `EXPECT_TRUE(cmpF(out_mat_gapi, out_mat_ocv));` and remove the rest.